package.json

```diff
@@ -50,8 +51,16 @@
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-react": "^7.12.3",
     "eslint-plugin-standard": "^4.0.0",
+    "husky": "^6.0.0",
+    "lint-staged": ">=10",
     "mocha": "^5.2.0",
     "prettier": "^1.15.3",
     "prettier-check": "^2.0.0"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --cache --fix",
+      "prettier --write"
```

Comment:
OK so I realized the problem with consecutive commits was caused by the addition of prettier here, as well as the inclusion of TWO eslint configuration files:

.eslintrc
.eslintrc.js

I removed the first .eslintrc file. This may have caused some unpredictable behavior with things like code editors — unclear which configuration file ESLint prefers. There's also the age old issue of a global install of eslint prevailing over the local installation. Who knows.

Finally, another reason for issues with consecutive commits contradicting each others styles had to do with the addition of `prettier`. the first eslint --fix command would "fix" things, then prettier would change things again (specifically single/double quotes), then the pre-commit hook would technically "pass", but it would set up the NEXT commit to automatically be invalid, leading to an endless cycle...

More reading on this for consideration in the future of our other repos' linting/general code style conundrums: https://www.futurehosting.com/blog/prettier-vs-eslint-whats-the-difference/